partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,193 @@
+# cli.context_class = StderrContext
+cli.context_settings = {
+    "show_default": True,
+    "auto_envvar_prefix": "PC",
+    # terminal_width
+    # max_content_width
+    # "": StderrContext,
+}
+
+
+@cli.result_callback()
+def process_result(result, verbose, quiet, no_ansi, package, format):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_result`

Add return type annotation: `None`

(ANN201)